server/sonar-server/src/test/java/org/sonar/server/usergroups/ws/UpdateActionTest.java

```diff
@@ -60,24 +61,27 @@
 
   @Rule
   public ExpectedException expectedException = ExpectedException.none();
-  DbClient dbClient = db.getDbClient();
-  DbSession dbSession = db.getSession();
-  GroupDao groupDao = dbClient.groupDao();
-  UserGroupDao userGroupDao = dbClient.userGroupDao();
 
-  PersistentSettings settings = mock(PersistentSettings.class);
-  WsTester ws = new WsTester(new UserGroupsWs(new UpdateAction(dbClient, userSession, new UserGroupUpdater(dbClient), settings)));
+  @Rule
+  public DefaultOrganizationProviderRule defaultOrganizationProvider = DefaultOrganizationProviderRule.create(db);
+
+  private DbClient dbClient = db.getDbClient();
+  private DbSession dbSession = db.getSession();
+  private PersistentSettings settings = mock(PersistentSettings.class);
+  private WsTester ws;
 
   @Before
   public void setUp() throws Exception {
+    GroupWsSupport groupSupport = new GroupWsSupport(db.getDbClient(), defaultOrganizationProvider);
+    ws = new WsTester(new UserGroupsWs(new UpdateAction(dbClient, userSession, groupSupport, settings)));
```

Comment:
In general, I find this is not a problem as I often use indentation to makes things more readable (even when line is not that long).

In this specific case:
* I would have used `WsActionTest` rather than `WsTester` (no need to instantiate `UserGroupWs`)
* I find it elegant to keep the `underTest` field that most, if not all, other UTs have
* I prefer to have a field for `DBClient` rather than duplicating `db.getDbClient().

This means that I would then write: 
```java
private DbClient dbClient = db.getDbClient();
private UpdateAction underTest = new UpdateAction(dbClient, new GroupWsSupport(dbClient, defaultOrganizationProvider), settings, defaultOrganizationProvider);
private WsActionTester actionTest = new WsActionTester(underTest);
```

which are not very long lines (so no indent even needed).

As you said, this is subjective. But I like to share such opinions of mine :)